tasks/senkevychs/01-dp.lisp

```diff
@@ -0,0 +1,60 @@
+(defparameter *possible-partitions* nil)
+
+;;; returns a list of all possible partitions of an input text
+(defun text-partition (text dictionary-path)
+  (setf *possible-partitions* nil)
```

Comment:
it would be better to handle this as `(let ((*possible-partitions* nil)) (move-forward ...))`. This will isolate access to the global variable in the current run from other threads.

But even better would be not to use the global variable at all and just pass it as a function call argument.